webpack/adminCss.config.js

```diff
@@ -35,6 +35,7 @@ module.exports = ( mode ) => ( {
 		'googlesitekit-wp-dashboard-css': './assets/sass/wpdashboard.scss',
 		'googlesitekit-authorize-application-css':
 			'./assets/sass/authorize-application.scss',
+		'googlesitekit-block-editor-css': './assets/sass/block-editor.scss',
```

Comment:
Let's move this to `webpack/gutenbergBlocks.config.js` similar to the approach taken in https://github.com/google/site-kit-wp/pull/10247 and https://github.com/google/site-kit-wp/pull/10191.

Let's also move the SCSS file to `blocks/reader-revenue-manager/block-editor-plugin/editor-styles.scss`.